### DISCUSSION TITLE
Prism.Regions

### DISCUSSION THREAD

**Discussion by WolfBullet** [PrismLibrary/Prism#3117] [WPF]
I'm working on a new application and trying to reuse some new strategies i'm learning from an older project. One of them is the use of regions and the region manager to bind views to other views in wpf. To be able to reuse the region manager since it auto resolves regions and seems easy to implement. But is it not available yet for .net 8.0. I also get following notification that is is unsupported.
![image](https://github.com/PrismLibrary/Prism/assets/73708188/1ba5f457-0bfa-4f62-9f64-ec5520c79764)

**Comment by brianlagunas** [comment]
Region manager is 1,000% supported in WPF. I imagine you have the wrong nuget references or namespaces declared. It will also work in .net 8 just fine. Hard to say what your issue is without a sample app

**Reply by WolfBullet** [reply]
we decided to simply use the view references avaialble in wpf but thank you. We noticed that it is an unnessery addon of abstraction for the scale of our project. But i will keep this in mind for further projects we ahve at our company.
